contrib/oath-toolkit/template.py

```diff
@@ -0,0 +1,44 @@
+pkgname = "oath-toolkit"
+pkgver = "2.6.11"
+pkgrel = 0
+build_style = "gnu_configure"
+make_cmd = "gmake"
+make_dir = "."
+hostmakedepends = [
+    "gmake",
+    "automake",
+    "libtool",
+    "gtk-doc-tools",
+    "pkgconf",
+    "libxml2-progs",
+]
+makedepends = ["linux-pam-devel", "libxml2-devel"]
+pkgdesc = "OATH one-time password toolkit"
+maintainer = "Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license = "GPL-3.0-or-later"
+url = "http://www.nongnu.org/oath-toolkit"
+source = f"http://download.savannah.gnu.org/releases/{pkgname}/{pkgname}-{pkgver}.tar.gz"
+sha256 = "fc512a4a5b46f4c43ab0586c3189fece4d54f7e649397d6fa1e23428431e2cb4"
+# TODO: failing checks are yet to be investigated
+options = ["!check"]
+
+
+def post_install(self):
+    for lib in ["pskc", "oath"]:
+        self.install_man(f"lib{lib}/man/{lib}_*.3", glob=True)
+        self.install_man(f"{lib}tool/{lib}tool.1")
```

Comment:
is there a reason the build system doesn't install these?

Reply:
I'm not sure why it doesn't, but it doesn't. Alpine for example also doesn't ship the man pages.

Reply:
Is there anything I should do regarding this? Or is it also okay to keep it like this?

Reply:
the same is done for the .3's?

Reply:
I called `apk manifest` on the packages and couldn't find the man pages, maybe I looked in the wrong place. Let me try again.

Reply:
yeah i removed this post_install and the same manpages are already installed

Reply:
alpine also has them https://pkgs.alpinelinux.org/contents?branch=edge&repo=community&arch=x86_64&page=2&name=oath%2dtoolkit%2ddoc

Reply:
> I called apk manifest on the packages and couldn't find the man pages,  

they are split to -man by default. for .3, it's -devel-man. in both cases, they autoinstall with some base- package when the respective one is installed

Reply:
yeah okay I apparently just checked the wrong packages, because the man pages are there.